modules/minimum-required-permissions-upi-azure.adoc

```diff
@@ -0,0 +1,241 @@
+* `Microsoft.Features/providers/features/register/action`
+====
+
+Following are the permissions required for deleting the {product-title} cluster in Azure public cloud.
+
+.Required permissions for deleting authorization resources
+[%collapsible]
+====
+* `Microsoft.Authorization/roleAssignments/delete`
+====
+
+.Required permissions for deleting compute resources
+[%collapsible]
+====
+* `Microsoft.Compute/disks/delete`
+* `Microsoft.Compute/galleries/delete`
+* `Microsoft.Compute/galleries/images/delete`
+* `Microsoft.Compute/galleries/images/versions/delete`
+* `Microsoft.Compute/virtualMachines/delete`
+* `Microsoft.Compute/images/delete`
+====
+
+.Required permissions for deleting identity management resources
+[%collapsible]
+====
+* `Microsoft.ManagedIdentity/userAssignedIdentities/delete`
+====
+
+.Required permissions for deleting network resources
+[%collapsible]
+====
+* `Microsoft.Network/dnszones/read`
+* `Microsoft.Network/dnsZones/A/read`
+* `Microsoft.Network/dnsZones/A/delete`
+* `Microsoft.Network/dnsZones/CNAME/read`
+* `Microsoft.Network/dnsZones/CNAME/delete`
+* `Microsoft.Network/loadBalancers/delete`
+* `Microsoft.Network/networkInterfaces/delete`
+* `Microsoft.Network/networkSecurityGroups/delete`
+* `Microsoft.Network/privateDnsZones/read`
+* `Microsoft.Network/privateDnsZones/A/read`
+* `Microsoft.Network/privateDnsZones/delete`
+* `Microsoft.Network/privateDnsZones/virtualNetworkLinks/delete`
+* `Microsoft.Network/publicIPAddresses/delete`
+* `Microsoft.Network/virtualNetworks/delete`
+====
+
+.Required permissions for checking health of resources
+[%collapsible]
+====
+* `Microsoft.Resourcehealth/healthevent/Activated/action`
+* `Microsoft.Resourcehealth/healthevent/Resolved/action`
+* `Microsoft.Resourcehealth/healthevent/Updated/action`
+====
+
+.Required permissions for deleting resource group
+[%collapsible]
+====
+* `Microsoft.Resources/subscriptions/resourcegroups/delete`
+====
+
+.Required permissions for deleting storage resources
+[%collapsible]
+====
+* `Microsoft.Storage/storageAccounts/delete`
+* `Microsoft.Storage/storageAccounts/listKeys/action`
+====
+
+[NOTE]
+====
+You can create a custom role with all the permissions and assign that custom role to the service principal. To create a custom role in the Azure portal, see link:https://learn.microsoft.com/en-us/azure/role-based-access-control/custom-roles[Azure custom roles] in the Azure documentation.
+====
+
+[NOTE]
+====
+For installing the {product-title} cluster, you must scope the permissions related to resource group creation, to a subscription level. After resource group is created, you can scope the rest of the permissions to the created resource group level. If the public DNS zone is present in a different resource group, then the network DNS zone related permissions must always be at subscription level.
+
+For deleting the {product-title} cluster, you can scope all the permissions to a subscription level.
```

Comment:
```suggestion
You can apply all the permissions to your subscription when deleting an {product-title} cluster.
```
Rewording suggestion

Reply:
Addressed

Reply:
For all the comments above, I am still wondering if `apply` will sound correct in place of `scope`. Technically `scope` is the appropriate term to use with permission.  @jinyunma wdyt ?

Reply:
Sounds good to me; we need to make similar changes to all the similar places then. cc @xenolinux

Reply:
Yes. I will make the changes. I was wondering if we can keep the sentence like this - 
`You can scope all the permissions to your subscription when deleting an {product-title} cluster.` Does this sound good?

Reply:
Addressed